frontend/lib/parsing/parsing-queries.cpp

```diff
@@ -100,25 +102,226 @@ static Parser helpMakeParser(Context* context,
   }
 }
 
+// <7F>HPECHPL
```

Comment:
Not even `CHAPEL!` 😄

Reply:
I was looking for a seven-byte thing to bring the total to a nice even 8 bytes. I guess I didn't consider punctuation! 😄